pynestml/frontend/pynestml_frontend.py

```diff
@@ -117,7 +126,13 @@ def install_nest(models_path, nest_path):
 
 
 def main():
-    """Returns the process exit code: 0 for success, > 0 for failure"""
+    """
+    Entry point for the command-line application.
+
+    Returns
+    -------
+    The process exit code: 0 for success, > 0 for failure
```

Comment:
Do we have/need a list of the exit codes > 0 and what they mean? Would it be good to have them here?

Reply:
Anything nonzero is typically interpreted as an error in POSIX. We only have 0 and 1, and I could just replace "> 0" with "1".